### PR TITLE
Rhpds workshops aap2 dev

### PR DESCRIPTION
**Pull request by rut31337**
##### SUMMARY
Add ability to download manifest from URL
Add RHPDS formatted output at end of provision

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

- provisioner

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: provisioner/provision_lab.yml [hunk 1]
```diff
@@ -201,5 +201,13 @@
           {{ demo_information }}
 
     - name: Print Summary Information
+      when: rhpds_info_output|default(False) == False
       debug:
         msg: "{{ summary_information }}"
+
+    - name: Print Summary Information
+      when: rhpds_info_output|default(False) == True
```
Comment (jkupferer) [inline provisioner/provision_lab.yml:209]:
```
    when: rhpds_info_output | default(false) | bool
```

File: provisioner/provision_lab.yml [hunk 2]
```diff
@@ -201,5 +201,13 @@
           {{ demo_information }}
 
     - name: Print Summary Information
+      when: rhpds_info_output|default(False) == False
```
Comment (jkupferer) [inline provisioner/provision_lab.yml:204]:
```
    when: not rhpds_info_output | default(false) | bool
```

File: roles/workshop_check_setup/tasks/controller.yml [hunk 3]
```diff
@@ -4,6 +4,17 @@
...
+  when:
+    - manifest_download_url is defined
+    - manifest_download_user is defined
+    - manifest_download_password is defined
+  ansible.builtin.get_url:
+    url: "{{ manifest_download_url }}"
+    dest: "{{ playbook_dir }}/manifest.zip"
+    username: "{{ manifest_download_user }}"
+    password: "{{ manifest_download_password }}"
+
```
Comment (jkupferer) [inline roles/workshop_check_setup/tasks/controller.yml:17]: I recommend:
```
- name: Download manifest.zip
  when:
    - manifest_download_url is defined
  block:
    - name: Enforce use of user/password for manifest download
      ansible.builtin.assert:
        that:
          - manifest_download_url.startswith('https://')
          - manifest_download_user is defined
          - manifest_download_password is defined
        fail_msg: >-
          manifest_download_url must be secure and requires manifest_download_user
          and manifest_download_password.

    - name: Download manifest.zip
        ansible.builtin.get_url:
          url: "{{ manifest_download_url }}"
          dest: "{{ playbook_dir }}/manifest.zip"
          username: "{{ manifest_download_user }}"
          password: "{{ manifest_download_password }}"
          force_basic_auth: true
```

**Review by jkupferer — APPROVED** [review]

**Comment by IPvSean** [comment]
@rut31337 you had two PRs that overlapped themselves, can you bebase and make sure it looks good?

**Comment by IPvSean** [comment]
@rut31337 bump this is stuck

**Comment by IPvSean** [comment]
please reopen when fixed @rut31337

**Pull request closed without merging**